src/output/human.rs

```diff
@@ -0,0 +1,286 @@
+//! An output format designed to be read by humans.
+
+use domain::{
```

Comment:
We’re (okay: I’m) generally not using nested use statements but have every module in its own line.